examples/flutter_gallery/lib/demo/customized/home.dart

```diff
@@ -0,0 +1,836 @@
+    _statsAnimationTwo = _initAnimation(
+        from: 0.01,
+        to: 1.0,
+        curve: Curves.easeOut,
+        controller: _statsAnimationControllerTwo);
+    _statsAnimationThree = _initAnimation(
+        from: 0.01,
+        to: 1.0,
+        curve: Curves.easeOut,
+        controller: _statsAnimationControllerThree);
+    _statsAnimationFour = _initAnimation(
+        from: 0.01,
+        to: 1.0,
+        curve: Curves.easeOut,
+        controller: _statsAnimationControllerFour);
+    _heartAnimation = _initAnimation(
+        from: 1.0,
+        to: 1.25,
+        curve: Curves.easeOut,
+        controller: _heartAnimationController);
+    _rotationAnimation = _initAnimation(
+        from: 0.0,
+        to: 0.5,
+        curve: Curves.easeOut,
+        controller: _rotationAnimationController);
+    _runnerFadeAnimation = _initAnimation(
+        from: 0.0,
+        to: 1.0,
+        curve: Curves.easeOut,
+        controller: _runnerAnimationController);
+    _numberCounterAnimation = _numberCounterAnimationController;
+  }
+
+  void _configureThemes() {
+    _targetPlatform = Theme.of(context).platform;
+    _platformTextAlignment = _targetPlatform == TargetPlatform.android
+        ? TextAlign.left
+        : TextAlign.center;
+    _themeData = ThemeData(
+      primaryColor: const Color(0xFF212024),
+      buttonColor: const Color(0xFF3D3D3D),
+      iconTheme: const IconThemeData(color: Color(0xFF4A4A4A)),
+      brightness: Brightness.light,
+      platform: _targetPlatform,
+    );
+  }
+
+  Widget _contentWidget() {
+    final double screenHeight = MediaQuery.of(context).size.height;
+    final TargetPlatform platform = Theme.of(context).platform;
+    final String backTitle =
+        platform == TargetPlatform.android ? 'TrackFit' : '';
+    return Scaffold(
+      backgroundColor: const Color(0xFF212024),
+      body: NotificationListener<Notification>(
+        onNotification: _handleScrollNotification,
+        child: GlowingOverscrollIndicator(
+          color: const Color(0x00FFFFFF),
+          axisDirection: AxisDirection.down,
+          showLeading: false,
+          showTrailing: false,
+          child: CustomScrollView(
+            controller: _scrollController,
+            physics: SnappingScrollPhysics(midScrollOffset: screenHeight),
+            shrinkWrap: true,
+            slivers: [
+              SliverAppBar(
+                pinned: false,
+                title: Text(
+                  backTitle,
+                ),
+                expandedHeight: screenHeight -
+                    _kDetailTabHeight -
+                    MediaQuery.of(context).padding.top,
+                leading: _buildBackButton(),
+                backgroundColor: const Color(0xFF212024),
+                flexibleSpace: FlexibleSpaceBar(
+                  background: _buildBody(),
+                ),
+              ),
+              SliverList(
+                delegate: SliverChildListDelegate(_stats),
```

Comment:
Is there just 1 thing in there? You can just use SliverToBoxAdapter and remove the SliverList, the SliverChildListDelegate and the List.